example/tutorial/tensorflow_example.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from mlflow import tensorflow, tracking
+import numpy as np
+import pandas as pd
+import shutil
+import tempfile
+import tensorflow as tf
+
+
+def main(argv):
+    # Builds, trains and evaluates a tf.estimator. Then, exports it for inference, logs the exported model 
+    # with MLflow, and loads the fitted model back as a PyFunc to make predictions.
+    (x_train, y_train), (x_test, y_test) = tf.keras.datasets.boston_housing.load_data()
+    feat_cols = [tf.feature_column.numeric_column(key="features", shape=(13,))]
+    feat_spec = {"features":tf.placeholder("float", name="features", shape=[102, 13])}
+    regressor = tf.estimator.DNNRegressor(
+    hidden_units=[50, 20],
+    feature_columns=feat_cols)
+    train_input_fn = tf.estimator.inputs.numpy_input_fn({"features": x_train}, y_train, batch_size=404, num_epochs=None, shuffle=True)
```

Comment:
Sorry, I randomly picked a batch size of 404 in the original script. Does it still give good results if we don't pass the batch size?

Reply:
Avg. loss for batch size of 404 is 30, avg. loss for default batch size is 34. Do you want to remove the batch size in this case?

Reply:
Yeah I think that's a small enough difference that we can remove it, thanks :)